.github/workflows/deploy-staging.yml

```diff
@@ -113,7 +113,6 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           DD_API_KEY: ${{secrets.DD_API_KEY}}
-          PROPERLY_ACCOUNT_API_SECRET: ${{secrets.PROPERLY_ACCOUNT_API_SECRET}}
```

Comment:
qq: This secret is being used by account-api repo [here](https://github.com/GoProperly/account-api/blob/84684584c5235bb56ad82a3ea63c5db32b62ed5d/serverless.yml#L172). Has that repo also been updated to fetching secrets using this the new method?

Reply:
Just saw the change is in progress:
- GoProperly/account-api#126